src/tjenesteomrader/pages/digitalTjeneste.md

```diff
@@ -0,0 +1,103 @@
+
+- Show & tell – vis fram innsikt, idéer og løsninger gjennom hele prosessen. Dette skaper eierskap og engasjement også utenfor teamet, og skaper rom for å få tilbakemeldinger og se avhengigheter produkter har
+  Vi har erfart at det er noen sentrale forutsetninger som må være på plass om man skal lykkes med utvikling av digitale produkter og tjenester:
+
+- Forankring og tillit hos ledelsen
+
+- Frihet og fred til å eksperimentere (det vi også kjenner som selvbestemmelse og autonomi)
+
+- Ansvar og myndighet til å ta beslutninger
+
+- Kontroll over innsatsfaktorene - du er mest sannsynlig avhengig av andre produkter og team i organisasjonen for å lykkes med produktet ditt
+
+- Dedikerte pilotkunder som faktisk tar i bruk det du lager i hverdagen og aksepterer at den er ufullstendig gjennom utviklingsløpet, men som bruker løsningen likevel
+
+Digital tjeneste- og produktutvikling er tett koblet sammen med strategi og kultur. Prinsippene gjelder for alle digitale tjenester og produkter, også datadrevet produkt- og tjenesteutvikling. De gjelder også for det vi kan kalle legacy systemer, eller der hvor det finnes eksisterende systemer som skal moderniseres, integreres eller kanskje erstattes med andre produkter eller tjenester.
+
+Vi kan bidra med hele produktteam eller dedikerte roller til produktteam som allerede finnes, som for eksempel:
+
+- Produktledelse
+- Tech lead eller arkitekt
+- Design lead
+- Utviklere og designere til team
+- Støttespillere, som for eksempel agile coach og scrum master
+
+Ikke bare er vi en gjeng flinke konsulenter med bred erfaring og kompetanse, vi har vært ute en vinterdag før og vi er modige, rause og læreglade. Eller folkelige, som noen ville ha kalt det. Vi tror sterkt på at å jobbe som en integrert del av virksomheten er avgjørende for å lykkes, enten om vi deltar som medlemmer i eksisterende team eller om vi representerer et produktteam. Vi tror det bidrar til å skape noe mer enn produktet eller tjenesten vi utvikler, vi skal tross alt utvikle og bidra i samfunnet og miljøene rundt oss.
+
+<div class=bigText>
+
+Ting vi er skikkelig gode på er:
+
+</div>
+
+- Innsikt, i form av intervjuer og observasjon, tjeneste- og brukerreiser og dataanalyse
+- Konseptutvikling, prototyping og testing
+- Design for digitale flater
+- Systemutvikling, fullstack
+- Sikkerhet og personvern
+- Test og kvalitet
+
+<div class="bigText">
+
+Andre ting du trenger for å utvikle gode digitale produkter og tjenester er:
+
+</div>
+
+- En retning og en plan. Vi kan bidra med [Strategiutvikling](./strategi)
+
+- Flinke folk som er dyktige på sine fagområder, som alltid vil lære mer og som er rause med sin kompetanse. Vi kan også bidra med [Datadrevet](./datadriv) produkt- og tjenesteutvikling
+
+- En kultur som er preget av tillit, trygghet og åpenhet. Vi kan bidra med Kulturutvikling.
+
+</div>
```

Comment:
Dette gjør jo markdown litt vanskeligere på en måte for å sørge for at alt er closed osv. Men jeg vet ikke om en umiddelbar bedre måte å løse det på, og du slipper jo i alle fall å kode opp lister osv her da.